### ISSUE TITLE
Depracated Magic Quotes keeping app from 

### ISSUE THREAD

**Issue by masantiago-wpi** [ucfopen/UDOIT#546]
Users in Canvas trying to access our UDOIT install are now getting this: 
`Deprecated: Function get_magic_quotes_gpc() is deprecated in /var/www/html/UDOIT-2.2.0/lib/ims-blti/OAuth.php on line 230`

I've been googling around this morning and at least one of the simpler to understand solutions (comment out any reference to the function) does not make things any better. This was working fine at this version and there hasn't been any changes to our php version (which is soon coming since we're on 7.1.3). Basically it worked at the end of spring term and now it's giving us this. 

I'm sure it's something basic i've missed. i'll keep looking but as i'm getting questions now, i figured i'd post this to see if anyone else recognizes.

**Comment by bagofarms** [comment]
I'm thinking your server has been upgraded to PHP 7.4, because this function wasn't deprecated until 7.4.0.  Could you confirm your server's PHP version?

**Comment by masantiago-wpi** [comment]
Hi, yes you're exactly right. I ended up finding that last night. It's a shared server and looks like it had been updated. This was also one of those things where the error the users saw on screen wasn't what was fully breaking the tool. I ended up needing to install a few missing php extensions after it was upgraded. It's working now. Will that deprecated function be an issue? Because of the shared space, i'm not sure i can downgrade to 7.3.
